eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -6,249 +40,195 @@ jobs:
   condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
   strategy:
     matrix:
-      # Build with no dependencies at all (No samples) - PRECONDITIONS ON
-      # WARNINGS_AS_ERRORS          ON
-      # TRANSPORT_CURL              OFF
-      # UNIT_TESTING                OFF
-      # UNIT_TESTING_MOCKS          OFF
-      # TRANSPORT_PAHO              OFF
-      # PRECONDITIONS               ON
-      # LOGGING                     OFF
-      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_NONE
-      Linux_ARM:
+      LinuxARM_Release_MapFiles_Logging:
         OSVmImage: 'ubuntu-18.04'
         vcpkg.deps: ''
+        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
         AptDependencies: 'gcc-arm-none-eabi'
-        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake'
+        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake -DPRECONDITIONS=OFF'
         AZ_SDK_C_NO_SAMPLES: 'true'
+        PublishMapFiles: 'true'
```

Comment:
How come we are still trying to publish or ARM? Did it work?

Reply:
It didn't fail, but didn't produce anything either. See the new PR, I removed map generation from ARM.